### ISSUE TITLE
Interface overhaul (bootstrap)

### ISSUE THREAD

**Issue by ifokkema** [LOVDnl/LOVD3#624]
After LOVD3.5 is out (multiple genome builds), it will be time for the long-wanted interface overhaul.
- Redo everything in bootstrap.
- Truly separate the HTML from the logic. Use proper templating or otherwise output buffering/rewriting.
- Redo forms completely, such that some checking will already be done by HTML5/JS such that it doesn't always require submission of the form.
- Consider doing more in ajax and modals; this may make things easier for us. E.g., the whole form doesn't need to be returned when there is an issue with the input; only the annotation of the errors can be returned or a positive status if everything succeeded. Note that the import should still be able to call `checkFields()`.
- Another request considering the forms; @leicray writes:
> Guidance when entering data into a new entry is provided in two ways. For some things, the guidance is presented immediately below the data-entry box. For other things, the guidance appears by hovering over the question mark (?) to the left of the data-entry box. That latter type of guidance can be quite extensive and helpful, but it disappears as soon as the cursor is moved from the question mark. It would be helpful to be able to "pin" the guidance open so that it can be read while entering data. The NCBI Gene pages allow "pinning" of information boxes, and several can be pinned open simultaneously.

- I *think* all of this should be able to be done in parts; i.e., I think this doesn't require a full rebuild that will take over a year. We can now and then add improvements whenever we get the time.

**Comment by leicray** [comment]
The ability to "pin" pop-up guidance would also ensure that guidance text could be copied and pasted if necessary.

**Comment by ifokkema** [comment]
> The ability to "pin" pop-up guidance would also ensure that guidance text could be copied and pasted if necessary.

Yes, the current implementation is not ideal at all and is also quite flaky.
